decidim-proposals/spec/system/proposals_fields_spec.rb

```diff
@@ -106,7 +106,7 @@
                    })
           end
 
-          let(:proposal_draft) { create(:proposal, :draft, users: [user], component: component, title: "More sidewalks and less roads", body: "He will not solve everything") }
+          let(:proposal_draft) { create(:proposal, :draft, users: [user], component: component, title: "More sidewalks and less roads", body: "They will not solve everything") }
```

Comment:
Isn't this one an example text? Not sure if it matters but it should probably be 
```suggestion
          let(:proposal_draft) { create(:proposal, :draft, users: [user], component: component, title: "More sidewalks and less roads", body: "It will not solve everything") }
```
since we're talking about a proposal, not a person 😁

Reply:
It makes sense!